book/_config.yml

```diff
@@ -21,7 +21,7 @@ parse:
     contact_email: "uwhackweek@uw.edu"
     dataset: "SnowEx"
     anon_reporting_url: "https://forms.gle/JcRfzraXraC8epFu9"
-    project_spreadsheet_url: "https://docs.google.com/spreadsheets/d/1oFcfWDGuDQB5muYM6pyQa7uDhoILV7knlBByLShDL18/edit?usp=sharing"
+    project_summary_url: "https://docs.google.com/document/d/15yGrmzGLW_HmaatZaZh-oySSKxl2qNPz6mIcy1V0liI/edit?usp=sharing"
```

Comment:
This is a personal preferences, but I always remove the added `?usp=sharing` from the URL. This is a Google tracking information and my approach is to not make it too easy on them :-)